exasol/ds/sandbox/templates/cloudformation/example-data/s3-bucket.jinja.yaml

```diff
@@ -1,8 +1,6 @@
 Resources:
   AiLabExampleDataBucket:
     Type: 'AWS::S3::Bucket'
-    Properties:
-      AccessControl: 'Private'
```

Comment:
Ah the bucket should be private, you can't change that

Reply:
The updated version of cf lint requests 
> 'AccessControl' is a legacy property.  Consider using 'AWS::S3::BucketPolicy' instead

See AWS docs [cfn-s3-bucket-accesscontrol](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-s3-bucket.html#cfn-s3-bucket-accesscontrol). 

> S3 buckets are created with ACLs disabled by default. Therefore, unless you explicitly set the [AWS::S3::OwnershipControls](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-s3-bucket-ownershipcontrols.html) property to enable ACLs, your resource will fail to deploy with any value other than Private. Use cases requiring ACLs are uncommon.

Reply:
If I understood the documents correctly, then `Properties: AccessControl: 'Private'` can and should be removed without changing the effective permissions. 

But maybe it would be wise to verify that, e.g. with our new Example-Data Bucket, as long as it is not used?

Reply:
I could verify privacy settings by deleting and re-deploying the cloudformation stack for the AI-Lab example data S3 bucket. Even after removing `Properties: AccessControl: 'Private' ` bucket still displays "_Block all public access_".